src/main/java/console/output/ConsolePrinter.java

```diff
@@ -0,0 +1,57 @@
+package console.output;
+
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+public class ConsolePrinter {
+
```

Comment:
This class have 6 public methods. I think that it can make some troubles. Contract of this class is too granulated. Consider creating DTO class, and pass it here. I don't want to know outside of this class how it is handled. I only want it to be printed.